R/approach_vaeac.R

```diff
@@ -185,23 +197,25 @@ setup_approach.vaeac <- function(internal, # add default values for vaeac here.
 #' @export
 #' @author Lars Henry Berge Olsen
 prepare_data.vaeac <- function(internal, index_features = NULL, ...) {
+  iter <- length(internal$iter_list)
+
+  n_coalitions <- internal$iter_list[[iter]]$n_coalitions
+  S <- internal$iter_list[[iter]]$S
+
   # If not provided, then set `index_features` to all non trivial coalitions
-  if (is.null(index_features)) index_features <- seq(2, internal$parameters$n_combinations - 1)
+  if (is.null(index_features)) index_features <- seq(2, n_coalitions - 1)
```

Comment:
Nit pick: I don't love if else without the curly brackets

Reply:
I generally agree, but these very short ones are OK I think. Keep.